builder/role_image_test.go

```diff
@@ -455,6 +455,8 @@ func TestBuildRoleImages(t *testing.T) {
 
 	err = roleImageBuilder.BuildRoleImages(
 		rolesManifest.Roles,
+		"test-registry.com:9000",
+		"test-organization",
```

Comment:
Retracting, with apologies, missed the code in `TestGetRoleDevImageName()` which does test this.